### DISCUSSION TITLE
Welcome to Car-Showroom-Management-System Discussions!

### DISCUSSION THREAD

**Discussion by Bhavesh-Kapur** [Bhavesh-Kapur/Car-Showroom-Management-System#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bhavesh-Kapur** [comment]
hello team members ;
we have to develop 
A CAR SHOWROOM MANAGEMENT SYSTEM 
any ideas would help

**Comment by Bhavesh-Kapur** [comment]
i have decided to build the project. in html css for the front end and django for the backend

**Comment by Bhavesh-Kapur** [comment]


**Reply by sidagl** [reply]
Okay

**Comment by Bhavesh-Kapur** [comment]
Dhruv hasn't send his github username , if somebody knows his username , share it

**Comment by Bhavesh-Kapur** [comment]
@sidagl you will be looking the frontend part 
for the demo day ie. this coming tuesday 
lets showcase the demo of our customer or client side frontend part 
you can start working on the client login page and client side frontend part 
Siddharth you can take help of @samariium with the ideas

**Reply by sidagl** [reply]
Okay cool, I will coordinate with @samariium for the same.

**Reply by samariium** [reply]
OK, i will help you and I also have an idea for this

**Comment by Bhavesh-Kapur** [comment]
Our project will require webpages like about page, brands webpage, contact , login page and a sign in page that would be integrated with the login page

you guys can start building the above pages and keep pushing the changes to the folder that i have created. frontend ideas.

**Reply by samariium** [reply]
I have an idea that firstly we make a main page that contains catalogue of all cars and in that we have a login page for customers and employees
Customers for booking and service 
Employees for all the work stuff

**Reply by sidagl** [reply]
Okay on it..

**Comment by Bhavesh-Kapur** [comment]
whats the update on the pages ??

**Comment by Bhavesh-Kapur** [comment]
any updates ??

**Comment by Bhavesh-Kapur** [comment]
I hope everyone had a great diwali weekend ,
now lets get back to work , finishing our project

**Comment by Bhavesh-Kapur** [comment]
create two different pages 
page 1: admin side portal having its dashboard , cars details where he can update it the details , bookings to show all the bookings.
page 2 client side where he can browse the cataloge and maybe book a test drive by giving its address details and all , and the bookings page where he can proceed with the booking                                   


remember to sync the repo before starting the work and always push the work once you leave your system 

put the html pages and their css js or any other files in the frontend ideas folder in the repo 
rename the files accordingly try not to have the same name of any other file 

keep me posted on the updations of the pages

**Reply by samariium** [reply]
Ok 👍

**Reply by sidagl** [reply]
okay cool

**Comment by samariium** [comment]
I'm done with client and admin side portal

**Comment by sidagl** [comment]
I have made major improvements to the admin portal page in collaboration with @samariium. @Bhavesh-Kapur kindly see and suggest if any changes are to be done.

**Comment by Bhavesh-Kapur** [comment]
Any update on the survey ??
the non technical and technical survey ?

**Comment by PrinceV007** [comment]
yes I have researched a bit on the page

**Comment by Aman-sain** [comment]
Bhavesh can u tell which part of the project is pending..so that i can help in it

**Reply by Bhavesh-Kapur** [reply]
backend part is still pending you can study it and start working on it

**Comment by Bhavesh-Kapur** [comment]
start working on the login logic for the backend

**Comment by DhruvTh2001** [comment]
@Bhavesh-Kapur i have started working for the backend form part for add car page

**Reply by PrinceV007** [reply]
there was some error in the linking of add car page i corrected it by redirecting it to the database

**Reply by Bhavesh-Kapur** [reply]
okay good

**Comment by Bhavesh-Kapur** [comment]
i have created the necessary models for customer

**Reply by DhruvTh2001** [reply]
i have created the addcustomer page to admin dashboard you can crosscheck and merge the request

**Reply by PrinceV007** [reply]
and i have linked the page you can also check and merge the same

**Comment by Bhavesh-Kapur** [comment]
any updates on the logic behind login ?? for both admin and customer logins??

**Reply by PrinceV007** [reply]
ok will look into it

**Comment by BhaviBhagwanani** [comment]
i have created the manageorders page for the dashboard you can now link it with backend

**Reply by Bhavesh-Kapur** [reply]
ok thanks

**Comment by Bhavesh-Kapur** [comment]
for the admin side portal view customers page is still missing anyone working on that ??
pls comment

**Comment by Aman-sain** [comment]
guys i had made a sign up page and catalog with car comparison tool but i need help in sync all of them so that they redirect if any error or add on plz let me know

**Reply by Bhavesh-Kapur** [reply]
okay the linking part will be done by the backend team

**Comment by Bhavesh-Kapur** [comment]
What's the status of the survey ?

**Comment by Bhavesh-Kapur** [comment]
Any updates regarding the survey ? what all questions decided so far ?

**Reply by samariium** [reply]
@sidagl working on that

**Reply by sidagl** [reply]
I have made a final draft of the survey and also shared it with you, any inputs and suggestions would be welcomed 
[https://forms.gle/k66wz4BUyP4FRaPWA](url)

**Reply by Bhavesh-Kapur** [reply]
@sidagl pls give access to the survey 

my gmail id is 
ddnbhavesh@gmail.com

**Comment by Bhavesh-Kapur** [comment]
@sidagl you have to change the survey format 

there would be two sections 
1st would be technincal and 
2nd would be non- technical  

two different headings and then their survey questions

**Reply by sidagl** [reply]
Okay on it...

**Comment by sidagl** [comment]
Good afternoon everyone
can everyone share their email IDs to add as collaborators in the survey form

**Reply by samariium** [reply]
svsamarsingh@gmail.com

**Reply by Aman-sain** [reply]
amansain29082001@gmail.com

**Comment by PrinceV007** [comment]
vibhavkhaneja2004@gmail.com

**Comment by PrinceV007** [comment]
Guys sab log wapas kab aa rahe ho jisse we can discuss the final steps to complete our project before the presentation
